cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
It is needed to print in the situation when perhaps format.json is locked unusually longer and user is not able to see what is going on with minio server.